defaults/main.yml

```diff
@@ -1,13 +1,19 @@
 ---
 marathon_version: "1.3.6"
 
+marathon_repo_host: repo.mesosphere.com
+
 # Debian: Mesosphere apt repository URL
-marathon_apt_package: "marathon={{ marathon_version }}-*"
-marathon_apt_repo: "deb http://repos.mesosphere.com/{{ansible_distribution|lower}} {{ansible_distribution_release|lower}} main"
+marathon_package_version: "1.0.540"
+marathon_os_distribution: "{{ ansible_distribution | lower }}"
+marathon_os_version: "{{ ansible_distribution_version.split('.') | join('') }}"
```

Comment:
(opt) more standard to replace chars: `ansible_distribution_version | regex_replace('\.', '')`

Reply:
Same comment as above.

Reply:
yeah you are right, this was taken from the (original) mesos role.. ok just keep as is 👍